tools/ci_build/build.py

```diff
@@ -217,9 +221,9 @@ def download_test_data(build_dir, src_url, expected_md5, azure_sas_key):
         if azure_sas_key:
             src_url += azure_sas_key
         if shutil.which('aria2c'):
-            subprocess.run(['aria2c','-x', '5', '-j',' 5',  '-q', src_url, '-d', cache_dir], check=True)
+            run_subprocess(['aria2c','-x', '5', '-j',' 5',  '-q', src_url, '-d', cache_dir])
```

Comment:
Don't change these two lines. As I'm not really want to output the the command. Because  the output contains azure_sas_key.

Reply:
I can change it back. Would like to point out that if it fails, the resulting subprocess.CalledProcessError shown in the build output contains the arguments. However, it looks like the Azure DevOps build output masks out the secret value.